firmware/README.md

```diff
@@ -444,6 +446,23 @@ The IM door sensors each have a sticker on them with their door IDs. On the bott
 - <The door ID converted to a decimal number> - if door ID was echoed to the cloud
 - -1 - if bad input was received and door ID was neither parsed or echoed to the cloud
 
+### **use_external_ble_antenna(String)**
+
+**Description:**
+
+Use this console function to toggle usage of the internal/external BLE antenna. The external antenna provides improved BLE signal compared to the internal antenna. However, using the external antenna when one is not physically attached to the Boron will result in much poorer signal.
+
+**Argument(s):**
+
+1. Enter 1 to use the external antenna
+2. Enter 0 to use the internal antenna
+
+**Return(s):**
+
+- 1 or 0, the value that was inputted
+- 1 or 0, the current selection of the antenna if 'e' is inputted
+- -1, invalid input detected
```

Comment:
Since the behaviour is pretty much the same as the `toggle_debugging_publishes` console function, could you please express the Arguments and Returns in the same way? I don't mind if you use your way in both or that you change yours to match the `toggle_debugging_publishes`. Either seem clear to me, I'd just like them to be consistent. Thanks!